src/WPF/ArcGISRuntime.WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -424,5 +456,45 @@ private void ResetCategories()
             // Set the expanded categories.
             SetExpandedCategories(expandedCategoryNames);
         }
+
+        private void ScreenshotButton_Click(object sender, RoutedEventArgs e)
+        {
+            GetJpgImage(SampleContainer);
+        }
+
+        private void GetJpgImage(UIElement source)
+        {
+            ContentControl sampleContainer = (ContentControl)source;
+            int actualHeight = (int)sampleContainer.ActualHeight;
+            int actualWidth = (int)sampleContainer.ActualWidth;
+
+            PresentationSource presentationSource = PresentationSource.FromVisual(source);
+
+            double scaleX, scaleY = 1;
+
+            scaleX = presentationSource.CompositionTarget.TransformToDevice.M11;
+            scaleY = presentationSource.CompositionTarget.TransformToDevice.M22;
+
+            int Height = (int)(source.DesiredSize.Height * scaleY);
+            int Width = (int)(source.DesiredSize.Width * scaleX);
+            System.Windows.Point relativePoint = source.PointToScreen(new System.Windows.Point(0d, 0d));
+            int X = (int)relativePoint.X;
+            int Y = (int)relativePoint.Y;
+            Bitmap screenshot = new Bitmap(Width, Height);
+            Graphics G = Graphics.FromImage(screenshot);
+            G.CopyFromScreen(X, Y, 0, 0, new System.Drawing.Size(Width, Height), CopyPixelOperation.SourceCopy);
+
+            // If scaling has occurred due to screen scaling we need to resize the image.
+            Bitmap resizedScreenshot = new Bitmap(screenshot, new System.Drawing.Size((int)(screenshot.Width / scaleX), (int)(screenshot.Height / scaleY)));
+
+            string sourcePath = ScreenshotManager.ScreenshotSettings.SourcePath;
+            string sampleName = _selectedSample.FormalName;
+            string categoryName = _selectedSample.Category;
+            string filePath = @$"{sourcePath}\{categoryName}\{sampleName}\{sampleName}.jpg"; // insert your filepath here to see the image output.
```

Comment:
I'm okay turning these 4 into a one-line call.
.NET Framework cannot use the `@$` string,  we have to use `$`.
```suggestion
            string filePath = $"{ScreenshotManager.ScreenshotSettings.SourcePath}\\{SampleManager.Current.SelectedSample.Category}\\{SampleManager.Current.SelectedSample.FormalName}\\{SampleManager.Current.SelectedSample.FormalName}.jpg";

```